symengine/sets.h

```diff
@@ -326,6 +367,23 @@ inline RCP<const Set> interval(const RCP<const Number> &start,
 }
 
 // ! \return RCP<const Set>
+inline RCP<const Set> imageset(const RCP<const Basic> &sym,
+                               const RCP<const Basic> &expr,
+                               const RCP<const Set> &base)
+{
+    if (not is_a<Symbol>(*sym))
```

Comment:
What if we need more that one symbol?
Like: `for x,y in [0,1] : x**2 + y**2`.
IMO this is doble.